langchain4j-mistral-ai/src/main/java/dev/langchain4j/model/mistralai/MistralAiChatModel.java

```diff
@@ -59,7 +62,7 @@ public class MistralAiChatModel implements ChatLanguageModel {
      * @param maxRetries   the maximum number of retries for API requests. It uses the default value 3 if not specified
      */
     @Builder
-    public MistralAiChatModel(String baseUrl,
+    private MistralAiChatModel(String baseUrl,
```

Comment:
why/is this change required?
same with all other classes below

Reply:
same comment. I saw that anthropic have it as private and I though that it's part of refactoring plan. If it's not I put back the builder factory as before